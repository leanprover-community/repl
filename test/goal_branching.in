{"cmd" : "theorem branching_test : 1 = 1 ∧ 2 = 2 ∧ 3 = 3 := by sorry", "sorries" : "individual"}

{"tactic": "constructor", "proofState": 0}

{"tactic": "exact rfl", "proofState": 1}

{"tactic": "constructor", "proofState": 2}

{"tactic": "exact rfl", "proofState": 3}

{"tactic": "exact rfl", "proofState": 4}
