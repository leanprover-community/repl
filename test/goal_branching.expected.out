{"sorries":
 [{"proofState": 0,
   "pos": {"line": 1, "column": 53},
   "parentDecl": "branching_test",
   "goals": ["⊢ 1 = 1 ∧ 2 = 2 ∧ 3 = 3"],
   "endPos": {"line": 1, "column": 58}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 22},
   "data": "declaration uses 'sorry'"}],
 "env": 0}

{"proofStatus": "Incomplete: open goals remain",
 "proofState": 1,
 "goals": ["case left\n⊢ 1 = 1", "case right\n⊢ 2 = 2 ∧ 3 = 3"]}

{"proofStatus": "Incomplete: open goals remain",
 "proofState": 2,
 "goals": ["case right\n⊢ 2 = 2 ∧ 3 = 3"]}

{"proofStatus": "Incomplete: open goals remain",
 "proofState": 3,
 "goals": ["case right.left\n⊢ 2 = 2", "case right.right\n⊢ 3 = 3"]}

{"proofStatus": "Incomplete: open goals remain",
 "proofState": 4,
 "goals": ["case right.right\n⊢ 3 = 3"]}

{"proofStatus": "Completed", "proofState": 5, "goals": []}

